nav2_velocity_smoother/README.md

```diff
@@ -59,14 +59,15 @@ velocity_smoother:
    	max_decel: [-2.5, 0.0, -3.2]  # Maximum deceleration, ordered [Ax, Ay, Aw]
    	odom_topic: "odom"  # Topic of odometry to use for estimating current velocities
    	odom_duration: 0.1  # Period of time (s) to sample odometry information in for velocity estimation
+	enable_stamped_cmd_vel: false
```

Comment:
Add a comment here about what this represents like the others

Reply:
b94181e0b8d6a393d49f3b8e56d8519e8e72c21b